src/audits/PageWithBadTitle.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.constants.Severity');
+
+axs.AuditRules.addRule({
+    name: 'pageWithBadTitle',
+    heading: 'The web page should have a title less than 66 characters long and does not contain bad characters',
+    url: 'https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#TODO-WIKIENTRY',
+    severity: axs.constants.Severity.WARNING,
+    relevantElementMatcher: function(element) {
+        return element.tagName.toLowerCase() == "title";
+    },
+    test: function(scope) {
+        // test length of title
+        if (document.title.length >= 66)
```

Comment:
`document` may not be what you expect here.

`scope` is an element which `relevantElementMatcher` returns true on (see https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/AuditRule.js#L243) and may be inside an iframe, in which case its owner document will be something other than `document` which will be the owner document of the context where this script is running. For example, if I inject this script into my page, `document` will be the top level document for my page, but `scope` may be inside an iframe which has its own `<title>`, which would be what I would want to check.

Since the element being passed in will be a `<title>`, you may want to rename `scope` to `title` or `titleEl` (like how https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/audits/ImageWithoutAltText.js calls the argument to its `test()` function `image`). Then you can check the `textContent` of the title element.